test/fixtures/full-project/app/router.js

```diff
@@ -7,7 +7,12 @@ const Router = EmberRouter.extend({
 });
 
 Router.map(function() {
+  this.route('angle-completion');
```

Comment:
This project already had a `router.js` file, but the route definitions were missing. So added them in here for us to list all routes.